lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -974,6 +974,8 @@ void LLVMToSPIRVBase::transVectorComputeMetadata(Function *F) {
   }
 }
 
+static void transMetadataDecorations(Metadata *MD, SPIRVEntry *Target) {
```

Comment:
Is this the right way to handle this?  Or should I move LLVMToSPIRVBase::transFPGAFunctionMetadata down to after the declaration of transMetadataDecorations?